drivers/usb/device/usb_dc_kinetis.c

```diff
@@ -147,8 +147,11 @@ static inline uint8_t get_bdt_idx(uint8_t ep, uint8_t odd)
 
 static int kinetis_usb_init(void)
 {
+#if (defined(FSL_FEATURE_SIM_OPT_HAS_USB_VOLTAGE_REGULATOR) &&                                     \
```

Comment:
I believe `IS_ENABLED(FSL_FEATURE_SIM_OPT_HAS_USB_VOLTAGE_REGULATOR)` is equivalent to what you achieved here.

Reply:
Unfortunately, IS_ENABLED doesn't work with values in parenthesis, which is the case for FSL_FEATURE_SIM_OPT_HAS_USB_VOLTAGE_REGULATOR. I can't change it since it is auto-generated code.